_posts/en/2017-03-13-performance-optimizations-1.md

```diff
@@ -0,0 +1,117 @@
+
+### Further information
+
+  * [Parallel script verification #2060](https://github.com/bitcoin/bitcoin/pull/2060)
+
+## Headers first synchronization
+Release: Bitcoin Core 0.10
+
+Striving to further improve initial block download time, the Core project introduced in late 2014 an important re-architecture of the mechanism used by nodes to synchronize with the “best chain”. 
+
+Initially, the process of bootstrapping a new Bitcoin client would involve a user fetching block data from a single peer with the consequence that any interruption or decrease in connection quality would significantly stall the process. With an ever-increasing blockchain size, this would result in sometimes massive waiting time for the synchronization to complete, with a large percentage of users reporting up to multi-day periods depending on their hardware.
+
+Headers first synchronization severely mitigates this issue using a new method that involves nodes first downloading and validating block headers from a single peer and then fetching block data in parallel from a multitude of others. 
+
+Complaints about initial block download time have been prevalent since the early days of Bitcoin. With headers first synchronization, the software took a major step forward in terms of usability for new users. Rather than wasting many hours on unreliable synchronization, nodes could now leverage their entire network of peers and cut down the bootstrapping time significantly. With the use of smarter algorithms, another asymptotic improvement was made to this crucial aspect of Bitcoin’s long term sustainability. 
+
+### Further information
+
+  * [Bitcoin-Qt 0.10.0 Release notes](https://bitcoin.org/en/release/v0.10.0#faster-synchronization)
+  * [Bitcoin.org Developer Guide](https://bitcoin.org/en/developer-guide#headers-first)
+  * [Pieter Wuille's post to the Bitcoin-dev mailing list](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2014-October/006724.html)
+  
+## Block file pruning
+Release: Bitcoin Core 0.11
+
+Pruning of old data was a concept first described by Satoshi Nakamoto in his white paper as a potential solution to disk space scarcity. Unfortunately, the original design was inadequate and could not be implemented as imagined by its creator. Seven years later, with the blockchain reaching more than a hundred gigabyte, the introduction of block file pruning as we know it today presents a major boon to users with limited resources.
+
+Block file pruning leverages previous work with ultraprune; users who have initially downloaded and validated the blockchain may now discard raw data older than 288 blocks. Because those nodes still preserve the full UTXO set, they remain able to validate unspent data, which is enough to provide full guarantees against double-spends. 
```

Comment:
I'd word this is "enough to fully validate new blocks" - just mentioning double-spends is understating it.